internal/controller/ngrok/kubernetesoperator_controller.go

```
@@ -132,7 +139,26 @@ func (r *KubernetesOperatorReconciler) SetupWithManager(mgr ctrl.Manager) error
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.18.4/pkg/reconcile
 func (r *KubernetesOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	return r.controller.Reconcile(ctx, req, new(ngrokv1alpha1.KubernetesOperator))
+	res, err := r.controller.Reconcile(ctx, req, new(ngrokv1alpha1.KubernetesOperator))
+	if err != nil || res.Requeue || res.RequeueAfter > 0 {
```

golangci-lint (2.12.2)

[error] SA1019: res.Requeue is deprecated: Use `RequeueAfter` instead. 

(staticcheck)